docs/catalog.md

```diff
@@ -171,7 +171,7 @@ You can use an action to store a document in a Cloudant database called `testdb`
 1. Store a document by using the `write` action in the package binding you created previously. Be sure to replace `/myNamespace/myCloudant` with your package name.
 
   ```
-  $ wsk action invoke /myNamespace/myCloudant/write --blocking --result --param dbname testdb --param doc '{"_id":"heisenberg", "name":"Walter White"}'
+  $ wsk action invoke /myNamespace/myCloudant/write --blocking --result --param dbname testdb --param doc "{\"_id\":\"heisenberg\",\"name\":\"Walter White\"}"
```

Comment:
yep, that's the plan. I don't know if it's a CLI problem, or just a Windows terminal problem.